pdbtools/pdb_tidy.py

```diff
@@ -134,7 +163,7 @@ def make_TER(prev_line):
         if line.startswith('ATOM'):
 
             is_gap = (int(line[22:26]) - int(prev_line[22:26])) > 1
-            if atom_section and (line[21] != prev_line[21] or is_gap):
+            if atom_section and (line[21] != prev_line[21] or (not strict and is_gap)):
```

Comment:
change `not strict` to the variable `not_strict`

Reply:
Done.